components/movementsensor/imuwit/imu.go

```diff
@@ -71,6 +89,7 @@ type wit struct {
 
 	mu sync.Mutex
 
+	port                    io.ReadWriteCloser
```

Comment:
You've added the port as an io.ReadWriteCloser to the imu struct, but is is being added when you initialise the function, look at what's happening in `gpsnmea/serial.go`, particularly lines 80-97 l[ink](https://github.com/viamrobotics/rdk/blob/10b2061145b82a5b8b67d00e2db1398ec7a5b1e4/components/movementsensor/gpsnmea/serial.go#L80).